language/move-prover/spec-lang/src/translate.rs

```diff
@@ -4105,6 +4186,28 @@ impl<'env, 'translator, 'module_translator> ExpTranslator<'env, 'translator, 'mo
         std::mem::replace(&mut self.subs, subs);
         result
     }
+
+    pub fn translate_from_move_value(value: &MoveValue) -> Value {
+        match value {
+            MoveValue::U8(n) => Value::Number(BigUint::from_u8(*n).unwrap()),
+            MoveValue::U64(n) => Value::Number(BigUint::from_u64(*n).unwrap()),
+            MoveValue::U128(n) => Value::Number(BigUint::from_u128(*n).unwrap()),
+            MoveValue::Bool(b) => Value::Bool(*b),
+            MoveValue::Address(a) => Value::Address(ModuleEnv::addr_to_big_uint(a)),
+            MoveValue::Signer(a) => Value::Address(ModuleEnv::addr_to_big_uint(a)),
+            MoveValue::Vector(vs) => {
+                let b = vs
+                    .iter()
+                    .map(|v| match v {
+                        MoveValue::U8(n) => *n,
+                        _ => unimplemented!("Not yet supported constant vector value: {:?}", v),
```

Comment:
Can this happen in reality? In that case we should do here `env.error` and not crash.

BTW, a good example if to report an error while I nearly never use static methods. As soon as you need more context for your functions you are stuck with static methods. If this would have a &self, you can add this kind of thing without any hassle.